Formula/zsh-lovers.rb

```diff
@@ -1,15 +1,13 @@
 class ZshLovers < Formula
   desc "Tips, tricks, and examples for zsh"
   homepage "https://grml.org/zsh/#zshlovers"
-  url "https://deb.grml.org/pool/main/z/zsh-lovers/zsh-lovers_0.9.1_all.deb",
-      :using => :nounzip
+  url "https://deb.grml.org/pool/main/z/zsh-lovers/zsh-lovers_0.9.1_all.deb"
   version "0.9.1"
   sha256 "011b7931a555c77e98aa9cdd16b3c4670c0e0e3b5355e5fd60188885a6678de8"
 
   bottle :unneeded
 
   def install
-    system "tar", "xf", "zsh-lovers_#{version}_all.deb"
```

Comment:
how is this not needed?

Reply:
Because a `deb` is automatically detected as `tar`.

Reply:
ah ok